### ISSUE TITLE
Windows: paths are not completely handled

### ISSUE THREAD

**Issue by adrientetar** [mbutterick/pollen#5]
I'm on Windows with latest racket and pollen.

What I did was `cd` to my working directory then ran this command:

```
"C:\Program Files\Racket\raco" pollen start
```

The `index.ptree` works, I can also access files in the folder that are not processed by pollen. However, when I click on `spec.html.pm`; I get this:

```
route-404: Can't find C:\Users\Adrien\Downloads\project\c\users\adrien\downloads\project\spec.html
```

Notice the stray path here.

Now `cd`'ing to racket directory and running it from here with a custom project path, I have basically the same error pattern but with racket path:

```
route-404: Can't find C:\Users\Adrien\Downloads\project\c\program files\racket\spec.html
```

Basically the same but with Racket's path.
For reference the command ran here was:

```
raco pollen start "C:\Users\Adrien\Downloads\project"
```

@mbutterick; Note that it's the first time I'm using Racket so I may be doing something wrong.
